client/app/vps/dashboard/vps-dashboard.controller.js

```diff
@@ -308,6 +304,24 @@ class VpsDashboardCtrl {
         this.ControllerHelper.navigation.getConstant("changeOwner").then(url => { this.actions.changeOwner.href = url; });
     }
 
+    getRegionsGroup (regions) {
+        this.regionsGroup = [];
+        if (regions) {
+            this.detailedRegions = !_.isArray(regions) ?
+                [this.RegionService.getRegion(regions)] :
+                _.map(regions, region => this.RegionService.getRegion(region));
+        }
+        this.regionsGroup = _.groupBy(this.detailedRegions, "country");
+    }
+
+    hasMultipleRegions () {
+        return _(this.detailedRegions).isArray() && this.detailedRegions.length > 1;
+    }
+
+    getActionStateParamString (params) {
+        if (!params) { return ""; }
```

Comment:
Could you avoid putting blocks on one line please 
Also you could replace this by a ternary expression ```return params ? `(${JSON.stringify(params)})` : ''```